tests/flows/teleporter/validator_churn.go

```diff
@@ -57,16 +59,56 @@ func ValidatorChurn(network *localnetwork.LocalNetwork, teleporter utils.Telepor
 	sentTeleporterMessage := sendEvent.Message
 
 	// Construct the signed warp message
-	signedWarpMessage := utils.ConstructSignedWarpMessage(ctx, receipt, subnetAInfo, subnetBInfo)
+	signedWarpMessage := utils.ConstructSignedWarpMessage(
+		ctx,
+		receipt,
+		subnetAInfo,
+		subnetBInfo,
+		nil,
+		network.GetSignatureAggregator(),
+	)
 
 	//
 	// Modify the validator set on Subnet A
 	//
 
 	// Add new nodes to the validator set
-	addValidatorsCtx, cancel := context.WithTimeout(ctx, 60*time.Second)
+	addValidatorsCtx, cancel := context.WithTimeout(ctx, 90*newNodeCount*time.Second)
```

Comment:
nit: test would be more robust if the timeout is based on the sleep period in the newNodeCount loop below (e.g., adding a sleep period constant, and changing 90 to (sleepPeriod + 85)).

Reply:
Made this change